clang/www/c_status.html

```diff
@@ -216,35 +216,31 @@ <h2 id="c2x">C23 implementation status</h2>
       </tr>
       <tr> <!-- Pre-Oct 2019 -->
         <td><a href="https://www.open-std.org/jtc1/sc22/wg14/www/docs/n2341.pdf">N2341</a></td>
-        <td class="unknown" align="center">Unknown</td>
-      </tr>
-      <tr> <!-- Pre-Oct 2019 -->
-        <td><a href="https://www.open-std.org/jtc1/sc22/wg14/www/docs/n2401.pdf">N2401</a></td>
-        <td class="unknown" align="center">Unknown</td>
+        <td class="none" align="center">No</td>
       </tr>
       <tr> <!-- Pre-Oct 2019 -->
         <td><a href="https://www.open-std.org/jtc1/sc22/wg14/www/docs/n2359.pdf">N2359</a></td>
         <td class="none" align="center">No</td>
       </tr>
       <tr> <!-- Nov 2020 -->
         <td><a href="https://www.open-std.org/jtc1/sc22/wg14/www/docs/n2546.pdf">N2546</a></td>
-        <td class="unknown" align="center">Unknown</td>
+        <td class="none" align="center">No</td>
       </tr>
       <tr>
         <td><a href="https://www.open-std.org/jtc1/sc22/wg14/www/docs/n2580.htm">N2580</a></td>
-        <td class="unknown" align="center">Unknown</td>
+        <td class="none" align="center">No</td>
       </tr>
       <tr> <!-- Apr 2021 -->
         <td><a href="https://www.open-std.org/jtc1/sc22/wg14/www/docs/n2640.htm">N2640</a></td>
-        <td class="unknown" align="center">Unknown</td>
+        <td class="none" align="center">No</td>
```

Comment:
This looks to be vacuously yes: you need both Annex F conformance (which clang doesn't claim yet) and non-0 `DEC_EVAL_METHOD` (which I think @tahonermann et al aren't planning on) for this to matter.

Reply:
Awesome, thanks!